src/lean_spec/subspecs/forkchoice/helpers.py

```diff
@@ -66,7 +72,9 @@ def get_fork_choice_head(
         current = max(children, key=lambda x: (vote_weights.get(x, 0), blocks[x].slot, x))
 
 
-def get_latest_justified(states: Dict[Bytes32, "State"]) -> Optional[Checkpoint]:
+def get_latest_justified(
+    states: Dict[Bytes32, "State"],
+) -> Optional[Checkpoint]:
```

Comment:
Are you using a linter that's not part of the repo setup? If so would be great if you could add as a separate PR